src/markdown-pages/create-installer/choosing-a-pv-provisioner.md

```diff
@@ -0,0 +1,75 @@
+---
+path: "/docs/create-installer/choosing-a-pv-provisioner"
+date: "2022-10-07"
+weight: 35
+linktitle: "Choosing a PV Provisioner"
+title: "Choosing a PV Provisioner"
+---
+
+kURL offers both [OpenEBS Local PV](/docs/add-ons/openebs) and [Rook](/docs/add-ons/rook) for dynamic provisioning of PersistentVolumes in a Kubernetes cluster. Each of these provisioners has benefits and limitations and should be considered depending on the size of the cluster and the characteristics of each workload consuming the volume.
+
+[OpenEBS Local PV](/docs/add-ons/openebs) provides dynamic PV provisioning of [Kubernetes Local Volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local). A local volume implies that storage is available only from a single node. As the Local Volume is accessible only from a single node, local volumes are subject to the availability of the underlying node and are not suitable for all applications. If a node becomes unhealthy, then the local volume will also become inaccessible, and a Pod using it will not be able to run. Applications using local volumes must be able to tolerate this reduced availability, as well as potential data loss, depending on the durability characteristics of the underlying disk. For more information about the OpenEBS, see the [add-on documentation](/docs/add-ons/openebs).
```

Comment:
I would split sentences into individual lines to make future diffs cleaner
```suggestion
kURL offers both [OpenEBS Local PV](/docs/add-ons/openebs) and [Rook](/docs/add-ons/rook) for dynamic provisioning of PersistentVolumes in a Kubernetes cluster.
Each of these provisioners has benefits and limitations and should be considered depending on the size of the cluster and the characteristics of each workload consuming the volume.

[OpenEBS Local PV](/docs/add-ons/openebs) provides dynamic PV provisioning of [Kubernetes Local Volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local).
A local volume implies that storage is available only from a single node.
As the Local Volume is accessible only from a single node, local volumes are subject to the availability of the underlying node and are not suitable for all applications.
If a node becomes unhealthy, then the local volume will also become inaccessible, and a Pod using it will not be able to run.
Applications using local volumes must be able to tolerate this reduced availability, as well as potential data loss, depending on the durability characteristics of the underlying disk.
For more information about the OpenEBS, see the [add-on documentation](/docs/add-ons/openebs).
```

Reply:
oops i wrote this in google docs. will do